From error27@gmail.com  Wed Feb 17 15:57:10 2010
From: Dan Carpenter <error27@gmail.com>
Date: Fri, 12 Feb 2010 14:48:01 +0300
Subject: staging: vt6655: add missing curly braces
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Jim Lieb <lieb@canonical.com>, Forest Bond <forest@alittletooquiet.net>, devel@driverdev.osuosl.org, kernel-janitors@vger.kernel.org
Message-ID: <20100212114801.GA5975@bicker>
Content-Disposition: inline


It's clear from the indent levels and the context that there are supposed to
be curly braces here.

Signed-off-by: Dan Carpenter <error27@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/vt6655/card.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- a/drivers/staging/vt6655/card.c
+++ b/drivers/staging/vt6655/card.c
@@ -2788,16 +2788,18 @@ void CARDvUpdateBasicTopRate (PVOID pDev
 
      //Determines the highest basic rate.
      for (ii = RATE_54M; ii >= RATE_6M; ii --) {
-         if ( (pDevice->wBasicRate) & ((WORD)(1<<ii)) )
+         if ( (pDevice->wBasicRate) & ((WORD)(1<<ii)) ) {
              byTopOFDM = ii;
              break;
+         }
      }
      pDevice->byTopOFDMBasicRate = byTopOFDM;
 
      for (ii = RATE_11M;; ii --) {
-         if ( (pDevice->wBasicRate) & ((WORD)(1<<ii)) )
+         if ( (pDevice->wBasicRate) & ((WORD)(1<<ii)) ) {
              byTopCCK = ii;
              break;
+         }
          if (ii == RATE_1M)
             break;
      }
